mantle/cmd/kola/testiso.go

```diff
@@ -738,7 +767,18 @@ func testLiveIso(ctx context.Context, inst platform.Install, outdir string, mini
 		liveConfig.AddFile(nmstateConfigFile, nmstateConfig, 0644)
 	}
 
-	mach, err := inst.InstallViaISOEmbed(nil, liveConfig, targetConfig, outdir, isOffline, minimal)
+	if isISOFromRAM {
+		isoKernelArgs = append(isoKernelArgs, "coreos.liveiso.fromram")
+	}
+
+	// Sometimes the logs that stream from various virtio streams can be
+	// incomplete because they depend on services inside the guest.
+	// When you are debugging earlyboot/initramfs issues this can be
+	// problematic. If you find yourself wanting more logging uncomment
+	// out the following to get it all on the console of the machine.
+	// isoKernelArgs = append(isoKernelArgs, "systemd.log_color=0 systemd.log_level=debug systemd.log_target=console")
```

Comment:
We could make this a CLI or environment variable too.

Reply:
made it a env var